resources/views/achievement/admin_includes/edit.blade.php

```diff
@@ -60,6 +60,42 @@
                 <label for="is_archived">Archive this achievement</label>
             </div>
 
+            <div class="form-group">
+                <input type="hidden" name="has_page" value="0">
+                <input type="checkbox" id="has_page" name="has_page"
+                       value="1" {{ ($achievement->has_page ? 'checked' : '') }}>
+                <label for="has_url">Can be achieved by visiting url</label>
+            </div>
+
+            <div id="achieve_page_block" @if(!$achievement->has_page) style="display: none;" @endif>
+
+                <div class="form-group">
+                    <label for="page_name">Achieve URL</label>
+                    <div class="input-group">
+                        <div class="input-group-prepend">
+                            <span class="input-group-text">saproto.nl/achieve/</span>
```

Comment:
maybe try getting that url dynamically. Very future proof if we would ever change domain

Reply:
I wouldn't be sure where to get the saproto.nl url, as I'd prefer to use that to keep the url short.